packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -333,12 +333,22 @@ async function migrateToArmAndMultiEnv(ctx: CoreHookContext): Promise<void> {
   await postMigration(projectPath, ctx, inputs, backupFolder);
 }
 
+async function getManifestPath(fx: string, projectPath: string): Promise<string> {
```

Comment:
Maybe we can first check `path.join(projectPath, AppPackageFolderName, REMOTE_MANIFEST)` and last check `path.join(fx, REMOTE_MANIFEST)` . If the three file path does not exist, may be we can throw an error.

Reply:
@XiaofuHuang  Why we need to first check `path.join(projectPath, AppPackageFolderName, REMOTE_MANIFEST)` ?
Because no matter what order, we always try to get a exist manifest file. 
Here is according to the version order. From old version to new version.